app/controllers/ops_controller/ops_rbac.rb

```diff
@@ -1138,7 +1137,7 @@ def rbac_build_myco_tree
         tag_node[:icon] = ActionController::Base.helpers.image_path('100/tag.png')
         tag_node[:select] = true if (@edit && @edit[:new][:filters].key?(tag_node[:key])) || (@filters && @filters.key?(tag_node[:key])) # Check if tag is assigned
         kids_checked = true if tag_node[:select] == true
-        cat_kids.push(tag_node)
+        cat_kids << tag_node
```

Comment:
no - that was an unintentional change.
I had changed it and meant to change back.
thanks